app-misc/ani-cli/ani-cli-4.5.ebuild

```diff
@@ -10,8 +10,8 @@ if [[ "${PV}" == *9999* ]]; then
 	inherit git-r3
 else
 	SRC_URI="
-		https://github.com/pystardust/${PN}/releases/download/v${PV}/ani-cli
-		https://github.com/pystardust/${PN}/releases/download/v${PV}/ani-cli.1
+		https://github.com/pystardust/${PN}/releases/download/v${PV}/ani-cli -> ani-cli-${PV}
```

Comment:
You can use just `${P}` which contains the package name and version. https://devmanual.gentoo.org/ebuild-writing/variables/#predefined-read-only-variables

Reply:
Thanks. Fixed !